content/en/docs/Writing policies/mutate.md

```diff
@@ -351,6 +351,190 @@ The anchor processing behavior for mutate conditions is as follows:
 
 2. Next, all tag-values without anchors and all add anchor tags are processed to apply the mutation.
 
+## Mutate Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports mutate existing resources with `patchesStrategicMerge` and `patchesJson6902`. Unlike standard mutate policies that are applied through admission review, mutate existing policies are applied in the background which updates existing resources in the cluster. Such policies are triggered via admission requests, and can be optionally configured to apply on policy updates.
+
+To define such a policy, trigger resources need to be specified in the `match` block. The target resources, resources that are mutated in the background, are specified in each mutate rule `mutate.targets`. Note that all target resources within a single rule must share the same definition schema. For example, a mutate existing rule fails if this rule mutates both `Pod` and `Deployment` as they do not share the same OpenAPI V3 schema (except `metadata`). 
+
+{{% alert title="Note" color="warning" %}}
+The proper permissions need to be granted to Kyverno service account in order to mutate existing resources. You will need to create a ClusterRole with right permissions and bind it to Kyverno service account via ClusterRoleBinding.
+{{% /alert %}}
+
+
+This policy applies when the trigger resource `staging/disctionary-1` ConfigMap changes, a label `foo=bar` will be added to the target resource `staging/secret-1` Secret.
```

Comment:
"This policy, which matches when the trigger resource named `dictionary-1` in the `staging` Namespace changes, writes a label `foo=bar` to the target resource named `secret-1` also in the `staging` Namespace."